invenio_communities/templates/communities/mycommunities_base.html

```diff
@@ -63,4 +77,4 @@
     <li><strong>Upload</strong> &mdash; get custom upload link to send to people</li>
 </ul>
 {% endif %}
-</div>
```

Comment:
I'm not removing it. This is a GitHub bug (see the complete file instead of the diff, it is still there). I only removed the newline at the end of the file.